developerportal/templates/molecules/cards/card-event.html

```diff
@@ -3,69 +3,32 @@
 {% load wagtailimages_tags %}
 {% load app_tags %}
 
-{% image resource.image width-1024 as image %}
-{% image resource.card_image width-1024 as card_image %}
-{% static "img/placeholders/event.jpg" as fallback_image %}
+{% image resource.image width-636 as image %}
+{% image resource.card_image width-636 as card_image %}
+{% static "img/placeholders/event-16-9.jpg" as fallback_image %}
 
-<a
-  href="{% pageurl resource %}"
-  class="card-link"
-  {% if resource.is_external %}
-    target="_blank"
-    rel="noopener noreferrer"
-  {% endif %}
-  data-type="{{ resource.resource_type }}"
->
-  <div class="card card-event{% if featured %} card-event-featured{% endif %}">
-    <div class="card-event-ribbon-mzp-l-content">
-      <div class="card-event-ribbon background-accent">
-        <span class="highlight2">{{ resource.event_dates|safe }}</span>
-      </div>
+<section class="mzp-c-card mzp-c-card-medium mzp-has-aspect-16-9">
```

Comment:
Also here, a specific reason to keep this a section? Generally, the `section` element is of very little value semantically unless it is being used to define an actual `region`. See this great writeup https://www.smashingmagazine.com/2020/01/html5-article-section/#so-what-about-lt-section-gt